From 4679051cd0ff68d902705fcba5b0ee7940fdbaa1 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sun, 30 Nov 2014 14:19:30 +0100
Subject: [PATCH] Remove Capacitors.pretty and Iut.pretty from the footprint
 library

They've been removed from GitHub and we don't have a snapshot :(
---
 template/fp-lib-table.for-github | 2 --
 template/fp-lib-table.for-pretty | 2 --
 2 files changed, 4 deletions(-)

diff --git a/template/fp-lib-table.for-github b/template/fp-lib-table.for-github
index fd9cd16..46bce09 100644
--- a/template/fp-lib-table.for-github
+++ b/template/fp-lib-table.for-github
@@ -5,7 +5,6 @@
   (lib (name Capacitors_SMD)(type Github)(uri ${KIGITHUB}/Capacitors_SMD.pretty)(options "")(descr "The way you like them."))
   (lib (name Capacitors_Tantalum_SMD)(type Github)(uri ${KIGITHUB}/Capacitors_Tantalum_SMD.pretty)(options "")(descr "The way you like them."))
   (lib (name Capacitors_ThroughHole)(type Github)(uri ${KIGITHUB}/Capacitors_ThroughHole.pretty)(options "")(descr "The way you like them."))
-  (lib (name Capacitors)(type Github)(uri ${KIGITHUB}/Capacitors.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Axial_ThroughHole)(type Github)(uri ${KIGITHUB}/Choke_Axial_ThroughHole.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Common-Mode_Wurth)(type Github)(uri ${KIGITHUB}/Choke_Common-Mode_Wurth.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Radial_ThroughHole)(type Github)(uri ${KIGITHUB}/Choke_Radial_ThroughHole.pretty)(options "")(descr "The way you like them."))
@@ -40,7 +39,6 @@
   (lib (name Inductors_NEOSID)(type Github)(uri ${KIGITHUB}/Inductors_NEOSID.pretty)(options "")(descr "The way you like them."))
   (lib (name Inductors)(type Github)(uri ${KIGITHUB}/Inductors.pretty)(options "")(descr "The way you like them."))
   (lib (name IR-DirectFETs)(type Github)(uri ${KIGITHUB}/IR-DirectFETs.pretty)(options "")(descr "The way you like them."))
-  (lib (name Iut)(type Github)(uri ${KIGITHUB}/Iut.pretty)(options "")(descr "The way you like them."))
   (lib (name Labels)(type Github)(uri ${KIGITHUB}/Labels.pretty)(options "")(descr "The way you like them."))
   (lib (name LEDs)(type Github)(uri ${KIGITHUB}/LEDs.pretty)(options "")(descr "The way you like them."))
   (lib (name Measurement_Points)(type Github)(uri ${KIGITHUB}/Measurement_Points.pretty)(options "")(descr "The way you like them."))
diff --git a/template/fp-lib-table.for-pretty b/template/fp-lib-table.for-pretty
index 4c3da38..edcb72d 100644
--- a/template/fp-lib-table.for-pretty
+++ b/template/fp-lib-table.for-pretty
@@ -5,7 +5,6 @@
   (lib (name Capacitors_SMD)(type KiCad)(uri ${KISYSMOD}/Capacitors_SMD.pretty)(options "")(descr "The way you like them."))
   (lib (name Capacitors_Tantalum_SMD)(type KiCad)(uri ${KISYSMOD}/Capacitors_Tantalum_SMD.pretty)(options "")(descr "The way you like them."))
   (lib (name Capacitors_ThroughHole)(type KiCad)(uri ${KISYSMOD}/Capacitors_ThroughHole.pretty)(options "")(descr "The way you like them."))
-  (lib (name Capacitors)(type KiCad)(uri ${KISYSMOD}/Capacitors.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Axial_ThroughHole)(type KiCad)(uri ${KISYSMOD}/Choke_Axial_ThroughHole.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Common-Mode_Wurth)(type KiCad)(uri ${KISYSMOD}/Choke_Common-Mode_Wurth.pretty)(options "")(descr "The way you like them."))
   (lib (name Choke_Radial_ThroughHole)(type KiCad)(uri ${KISYSMOD}/Choke_Radial_ThroughHole.pretty)(options "")(descr "The way you like them."))
@@ -40,7 +39,6 @@
   (lib (name Inductors_NEOSID)(type KiCad)(uri ${KISYSMOD}/Inductors_NEOSID.pretty)(options "")(descr "The way you like them."))
   (lib (name Inductors)(type KiCad)(uri ${KISYSMOD}/Inductors.pretty)(options "")(descr "The way you like them."))
   (lib (name IR-DirectFETs)(type KiCad)(uri ${KISYSMOD}/IR-DirectFETs.pretty)(options "")(descr "The way you like them."))
-  (lib (name Iut)(type KiCad)(uri ${KISYSMOD}/Iut.pretty)(options "")(descr "The way you like them."))
   (lib (name Labels)(type KiCad)(uri ${KISYSMOD}/Labels.pretty)(options "")(descr "The way you like them."))
   (lib (name LEDs)(type KiCad)(uri ${KISYSMOD}/LEDs.pretty)(options "")(descr "The way you like them."))
   (lib (name Measurement_Points)(type KiCad)(uri ${KISYSMOD}/Measurement_Points.pretty)(options "")(descr "The way you like them."))
-- 
2.1.0

